modules/end-user-guide/pages/mounting-configmaps-and-secrets.adoc

```diff
@@ -0,0 +1,106 @@
+:navtitle: Mounting ConfigMaps and Secrets
+:keywords: user-guide, configuring, user, configmaps, configmap, secret, secrets, mounting, mount
+:page-aliases:
+
+[id="mounting-configmaps-and-secrets_{context}"]
+= Mounting ConfigMaps and Secrets
+
+ConfigMaps are {platforms-name} objects that store non-confidential data such as configuration values for an application.
+
+Secrets are {platforms-name} objects that store confidential data such as user names, passwords, authentication tokens, and configurations in an encrypted form.
```

Comment:
```suggestion
Secrets are {orch-name} objects that store confidential data such as user names, passwords, authentication tokens, and configurations in an encrypted form.
```

Reply:
@dkwon17, this is already fixed in my counter-PR. Feel free to resolve this thread.